textrepo-app/src/main/java/nl/knaw/huc/TextRepoApp.java

```diff
@@ -93,6 +97,18 @@ public void run(TextRepoConfiguration config, Environment environment) {
     final Supplier<UUID> uuidGenerator = UUID::randomUUID;
 
     var jdbi = createJdbi(config, environment);
+
+    var dataSourceFactory = config.getDataSourceFactory();
+    var configure = Flyway
```

Comment:
naamgeving zelfst.naamw.? "config" of "flywayConfig" oid?

Reply:
Opgelost in 0446433